sql/ddl_api.sql

```diff
@@ -216,3 +216,6 @@ CREATE OR REPLACE PROCEDURE @extschema@.refresh_continuous_aggregate(
     force                    BOOLEAN = FALSE
 ) LANGUAGE C AS '@MODULE_PATHNAME@', 'ts_continuous_agg_refresh';
 
+CREATE OR REPLACE PROCEDURE @extschema@.move_hypertable_invalidations(
```

Comment:
Not sure about having this function in the extension schema (typically public). It feels like an "internal" function or a developer function. I think other cagg API functions were put int timescaledb_functions schema, right? Maybe this one fits better there.

Reply:
This is now moved to pull request #8141 as agreed on.